src/pages/splash/splash.less

```diff
@@ -248,6 +248,18 @@
   }
 
   /**
+   * Maker Party Promo
+   */
+  #mp-promo {
+    .promo {
+      .promoMixin(@vikingYellow);
```

Comment:
From the [guideline](https://github.com/mozilla/webmaker-desktop/issues/79#issuecomment-113186712) it feels like this will be applied to any future promo banners, not just the Maker Party one we have in this PR? :eyes: